### ISSUE TITLE
Build fails with neovim 0.5.0

### ISSUE THREAD

**Issue by hsanson** [tweekmonster/vim-testbed#77]
Trying to build latest released neovim v0.5.0 results in an error:

```
-- Found LibLUV: /vim/neovim-neovim/neovim-v0.5.0/.deps/usr/lib/libluv.a (Required is at least version "1.30.0")                                  
CMake Error at cmake/LibFindMacros.cmake:263 (message):                                                                                           
  REQUIRED PACKAGE NOT FOUND                                                                                                                      
                                                                                                                                                  
  We could not find development headers for TreeSitter.  Do you have the                                                                          
  necessary dev package installed? This package is REQUIRED and you need to
  install it or adjust CMake configuration in order to continue building
  nvim.                   
                                    
  Relevant CMake configuration variables:
                                    
    TreeSitter_INCLUDE_DIR=<not found>
    TreeSitter_LIBRARY=<not found>

  You may use CMake GUI, cmake -D or ccmake to modify the values.  Delete 
  CMakeCache.txt to discard all values and force full re-detection if
  necessary.

Call Stack (most recent call first): 
  cmake/FindTreeSitter.cmake:11 (libfind_process)
  CMakeLists.txt:393 (find_package)
```

Command used:

```
install_vim -tag neovim:v0.5.0 -build
```

**Comment by hsanson** [comment]
Adding -DUSE_BUNDLED_TS=ON to DEPS_CMAKE_FLAGS seems to solve the problem and neovim 0.5 is able to compile. Also note that this flag is present in the insall_vim script in master but not in the "latest" docker image available in docker hub.

**Comment by blueyed** [comment]
Thanks for the issue.  I've quickly started looking into this, but it appears to work with Neovim master at least apparently!? (Build: https://app.travis-ci.com/github/blueyed/vim-testbed/builds/233557830, via https://github.com/Vimjas/vim-testbed/pull/78 - which currently still fails on GitHub Actions, where I will move CI to eventually (via https://github.com/Vimjas/vim-testbed/tree/ci-gha)).

> Adding -DUSE_BUNDLED_TS=ON to DEPS_CMAKE_FLAGS seems to solve the problem and neovim 0.5 is able to compile. Also note that this flag is present in the insall_vim script in master but not in the "latest" docker image available in docker hub.

Do you mean `DEPS_CMAKE_FLAGS` is not available in `/sbin/install_vim`?
Make sure to have pulled the latest image then, since the contents appears to be the same as on the master branch for me (Digest: sha256:5a55710f70f20e6a45aa07bcf5aaac58c5545c90f7fdc1eaaf55790cf5318d78)

I will investigate further in the next days, in particular trying to test the 0.5.0 tag explicitly (since it is a new / the latest stable release).

**Comment by hsanson** [comment]
> Digest: sha256:5a55710f70f20e6a45aa07bcf5aaac58c5545c90f7fdc1eaaf55790cf5318d78

Where can I find this image? it is not found in Docker Hub as far as I can tell.

I am using this image:

- https://hub.docker.com/r/tweekmonster/vim-testbed

Latest build is from 2 years ago and does not have the `-DUSE_BUNDLED_TS=ON` configuration added to compilation flags.

**Comment by blueyed** [comment]
The repo on Docker Hub is: https://hub.docker.com/r/testbed/vim

**Comment by blueyed** [comment]
This issue in particular was fixed in 3852e69 already.. :)
